### ISSUE TITLE
DatePicker accessibility issues

### ISSUE THREAD

**Issue by betrue-final-final** [OfficeDev/office-ui-fabric-react#1601]
Environment: IE11 and JAWS 18
Are you willing to submit a PR to fix? No. I need help
Requested priority: High

1. Doesn't announce itself as a date picker, just "Not in a table, row selected"
Expected: Should say "Select a date" along with hints on how to get to the controls to do this.

2. Navigating around the month just repeats "Select the first day of the week"
Expect: Should announce the day that it's on

3. Tabbing into the year/month side just says "Out of table, 2017"
Expected: should say "Select a year, 2017 selected" and hint text.

4. Month buttons just say the acronym, which is the visible label.
Expect: Say the full month name

**Comment by scriby** [comment]
1. We should try to add a summary to the table and see if it resolves it.

2. I just tested the date picker in JAWS + Chrome and didn't encounter this problem.

**Comment by betrue-final-final** [comment]
Try it again on JAWS and IE. Chrome has issues with JAWS and IE is what blind users use.

________________________________

**Comment by scriby** [comment]
I just tested in IE11 + JAWS and it read the days of the month when moving around the table. I am using latest fabric-react (2.22.0).

**Comment by betrue-final-final** [comment]
I’ll double check.

**Comment by betrue-final-final** [comment]
Ok here’s a picture. It says “Jan” not January:
[cid:image001.jpg@01D2BDEF.D4991000]

**Comment by betrue-final-final** [comment]
All issues above are not repro in Narrator. However the "Go to Today" button just says "Group".

**Comment by kellygorr** [comment]
Go to today button tested with narrator and appears to work in this current version.

**Comment by suztocco** [comment]
Please see related bug on this. https://github.com/OfficeDev/office-ui-fabric-react/issues/3250
#3250

**Comment by betrue-final-final** [comment]
All these issues seem to be resolved.
